geode-core/src/main/java/org/apache/geode/internal/cache/tx/DistributedTXRegionStub.java

```diff
@@ -120,12 +137,11 @@ public boolean containsKey(KeyInfo keyInfo) {
     }
   }
 
-
   @Override
   public boolean containsValueForKey(KeyInfo keyInfo) {
     try {
-      RemoteContainsKeyValueResponse response = RemoteContainsKeyValueMessage
-          .send((InternalDistributedMember) state.getTarget(), region, keyInfo.getKey(), true);
+      RemoteContainsKeyValueResponse response = sendRemoteContainsKeyValueMessage(
+          (InternalDistributedMember) state.getTarget(), keyInfo.getKey());
```

Comment:
The method lost valueCheck (true) value with the current method implementation.
RemoteContainsKeyValueResponse send(InternalDistributedMember recipient,
      LocalRegion r, Object key, boolean valueCheck)